Robot Code/src/org/usfirst/frc/team3316/robot/commands/climbing/MoveUp.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team3316.robot.commands.climbing;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+public class MoveUp extends DBugCommand
+{
+
+	private double speed, initAngle;
+	private boolean onTarget;
+	
+	public MoveUp() {
+		requires(Robot.climbing);
+	}
+
+	protected void init()
+	{
+		initAngle = Robot.climbing.getAngle();
+	}
+
+	protected void execute()
+	{
+		speed = (double) config.get("climbing_Speed");
+
+		isFin = !Robot.climbing.setMotors(speed);
+
+		onTarget = Robot.climbing
+				.getAngle() >= ((double) config.get("CLIMBING_POT_THRESHOLD")
+						+ initAngle);
```

Comment:
Why would we want to change this? This indicates the winch has reached the correct height, no? This should be a constant.